app/src/main/java/org/akvo/flow/util/logging/RavenEventBuilderHelper.java

```diff
@@ -37,6 +41,10 @@ public RavenEventBuilderHelper(Map<String, String> tags) {
     @Override
     public void helpBuildingEvent(EventBuilder eventBuilder) {
         eventBuilder.withSdkName(RavenEnvironment.SDK_NAME + ":android");
+        String release = tags.get(VERSION_NAME_TAG_KEY);
```

Comment:
I have moved it inside the loop. For the second concern: I need both, it is just for practicalities, release info appears in sentry page: https://sentry.io/akvo-foundation/flow-android/releases/ but it is still practical to have the tag inside the issue so you can click it and see all the issues for that tag..